RetailCoder.VBE/Extension.cs

```diff
@@ -70,6 +70,7 @@ public void OnStartupComplete(ref Array custom)
         public void OnDisconnection(ext_DisconnectMode RemoveMode, ref Array custom)
         {
             _app.Shutdown();
+            _app.Dispose();
```

Comment:
`_app` is instantiated like this:

    _app = _kernel.Get<App>();

Therefore, the kernel should be responsible for disposing it. This line isn't needed.

If `App.Dispose` isn't running when `_kernel.Dispose` is called, then forcing `App.Dispose` manually is only hiding other problems.

Now, looking at [RubberduckModule.cs](https://github.com/rubberduck-vba/Rubberduck/blob/next/RetailCoder.VBE/Root/RubberduckModule.cs) it seems `App` isn't actually handled by the kernel. Adding an `InSingletonScope()` binding for it (geez, how did I miss that!?) would make it dispose correctly.